specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-11-01-preview/examples/CreateSparkCluster.json

```diff
@@ -0,0 +1,323 @@
+            "ossVersion": "2.2.3",
+            "components": [
+              {
+                "name": "HDFS",
+                "version": "2.2.3"
+              }
+            ],
+            "identityProfile": {
+              "msiResourceId": "/subscriptions/subid/resourceGroups/hiloResourcegroup/providers/Microsoft.ManagedIdentity/userAssignedIdentities/test-msi",
+              "msiClientId": "de91f1d8-767f-460a-ac11-3cf103f74b34",
+              "msiObjectId": "40491351-c240-4042-91e0-f644a1d2b441"
+            },
+            "authorizationProfile": {
+              "userIds": [
+                "testuser1",
+                "testuser2"
+              ]
+            },
+            "serviceConfigsProfiles": [
+              {
+                "serviceName": "spark-service",
+                "configs": [
+                  {
+                    "component": "spark-config",
+                    "files": [
+                      {
+                        "fileName": "spark-defaults.conf",
+                        "values": {
+                          "spark.eventLog.enabled": "true"
+                        }
+                      }
+                    ]
+                  }
+                ]
+              },
+              {
+                "serviceName": "yarn-service",
+                "configs": [
+                  {
+                    "component": "yarn-config",
+                    "files": [
+                      {
+                        "fileName": "core-site.xml",
+                        "values": {
+                          "fs.defaultFS": "wasb://testcontainer@teststorage.dfs.core.windows.net/",
+                          "storage.name": "teststorage",
+                          "storage.protocol": "wasb",
+                          "storage.key": "test key",
+                          "storage.container": "testcontainer"
+                        }
+                      },
+                      {
+                        "fileName": "yarn-site.xml",
+                        "values": {
+                          "yarn.webapp.ui2.enable": "false"
+                        }
+                      }
+                    ]
+                  }
+                ]
+              }
+            ],
+            "connectivityProfile": {
+              "web": {
+                "fqdn": "cluster1.clusterpool1.westus2.projecthilo.net"
+              },
+              "ssh": [
+                {
+                  "endpoint": "cluster1.clusterpool1.westus2.projecthilo.net/ssh/host/sshnode-0"
+                },
+                {
+                  "endpoint": "cluster1.clusterpool1.westus2.projecthilo.net/ssh/host/sshnode-1"
+                }
+              ]
+            },
+            "sparkProfile": {},
+            "sshProfile": {
+              "count": 2,
+              "podPrefix": "sshnode"
+            },
+            "rangerPluginProfile": {
+              "enabled": false
```

Comment:
>enabled [](http://example.com/codeflow?start=15&length=7)

Is enabled be false by default if a rangerPluginProfile is not specified, making this addition optional?

Reply:
(do you even need the enable property? Would anyone createa  ranger plugin and disable it?)

Reply:
Yes, it is optional, so it is false by default. I think we can change it true here or remove this